src/sentry/integrations/gitlab/webhooks.py

```diff
@@ -52,6 +57,20 @@ def get_repo(self, integration, organization, event):
             return None
         return repo
 
+    def update_repo_data(self, repo, event):
+        """
+        Given a webhook payload, update stored repo data.
+
+        Assumes a 'project' key in event payload, with certain subkeys. Rework
+        this if that stops being a safe assumption.
+        """
+
+        project = event['project']
+        repo.name = '{} / {}'.format(project['namespace'], project['name'])
```

Comment:
```suggestion
        repo.name = u'{} / {}'.format(project['namespace'], project['name'])
```

To prevent potential UnicodeErrors